scripts/CMakeLists.txt

```diff
@@ -65,29 +65,23 @@ SET (SCRIPTS
 	TrainingMission
 	Y2K)
 
-SET (SCRIPTTARGETS )
-
-FOREACH( SCRIPT ${SCRIPTS})
+FOREACH(SCRIPT ${SCRIPTS})
  
-  ADD_LIBRARY(${SCRIPT} SHARED ${CPPS} "${SCRIPT}.cpp")
-  set_target_properties(${SCRIPT} PROPERTIES PREFIX "")
-  SET(SCRIPTTARGETS ${SCRIPTTARGETS} ${SCRIPT})
-    IF(UNIX)
-      GET_TARGET_PROPERTY(SCRIPT_LIB ${SCRIPT} LOCATION)
-      ADD_CUSTOM_COMMAND(
-	TARGET ${SCRIPT}
-	POST_BUILD
-	COMMAND ${CMAKE_COMMAND} -E copy ${SCRIPT_LIB} "${CMAKE_SOURCE_DIR}/scripts/data/linuxfullhog/"
-	COMMENT "Copying file ${SCRIPT_LIB} to hogfile directory..."
-	)
-      ADD_CUSTOM_COMMAND(
-	TARGET ${SCRIPT}
-	POST_BUILD
-	COMMAND ${CMAKE_COMMAND} -E copy ${SCRIPT_LIB} "${CMAKE_SOURCE_DIR}/scripts/data/linuxdemohog/"
-	)
-
-    ENDIF()
-
+	ADD_LIBRARY(${SCRIPT} SHARED ${CPPS} "${SCRIPT}.cpp")
+	set_target_properties(${SCRIPT} PROPERTIES PREFIX "")
+	IF(UNIX)
+		ADD_CUSTOM_COMMAND(
+			TARGET ${SCRIPT}
+			POST_BUILD
+			COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:${SCRIPT}> "${CMAKE_SOURCE_DIR}/scripts/data/linuxfullhog/"
```

Comment:
I think we'll have to revisit this, copying into the source directory is not very clean, this should probably be the binary one, then some extra code to deal with whatever needs to happen to those files later (install them?)

Reply:
Yeah, I was thinking on it. Will rework this some day.